### DISCUSSION TITLE
Lists enhancement

### DISCUSSION THREAD

**Discussion by yasinnaal** [community/community#59955] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello Team,

I suggest:

- Create sperate tab for lists (just like projects, repo, …)
- Sperate list creation from starred totally. 
- Option to add repo from List (edit - list) 

Please upvote if you found it useful.
 
Thanks

**Comment by PlatinumDave1** [comment]
> Gusta que ![README.md](https://github.com/community/community/files/11951055/README.md)

**Reply by kaschey77** [reply]
Hello

**Comment by Joethanyakor** [comment]
> > Gusta que ![README.md](https://github.com/community/community/files/11951055/README.md)
> 
> 

Yes

**Comment by Joethanyakor** [comment]
> > Gusta que ![README.md](https://github.com/community/community/files/11951055/README.md)
> 
>

**Reply by Joethanyakor** [reply]
Hi

**Comment by Joethanyakor** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> 
> 
> Hello Team,
> 
> 
> 
> I suggest:
> 
> 
> 
> - Create sperate tab for lists (just like projects, repo, …)
> 
> - Sperate list creation from starred totally. 
> 
> - Option to add repo from List (edit - list) 
> 
> 
> 
> Please upvote if you found it useful.
> 
>  
> 
> Thanks
> 
> 
> 
>

**Comment by Obmo7994** [comment]
_int @sous-marin

**Comment by Obmo7994** [comment]
> > ### Select Topic Area
> > 
> > Product Feedback
> > 
> > ### Body
> > 
> > 
> > 
> > Hello Team,
> > 
> > 
> > 
> > I suggest:
> > 
> > 
> > 
> > - Create sperate tab for lists (just like projects, repo, …)
> > 
> > - Sperate list creation from starred totally. 
> > 
> > - Option to add repo from List (edit - list) 
> > 
> > 
> > 
> > Please upvote if you found it useful.
> > 
> >  
> > 
> > Thanks
> > 
> > 
> > 
> > 
> 
> 

Int printf [lapte wibe ](secratche
)

**Comment by MohammedShamimMridha** [comment]
@yasinnaal

**Comment by m7mad120059** [comment]
Hh

**Comment by Isa232323** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> 
> Hello Team,
> 
> I suggest:
> 
> - Create sperate tab for lists (just like projects, repo, …)
> - Sperate list creation from starred totally. 
> - Option to add repo from List (edit - list) 
> 
> Please upvote if you found it useful.
>  
> Thanks
> 
>

**Comment by goalend** [comment]
Y

**Comment by m7mad120059** [comment]
I

**Reply by goalend** [reply]
[]()

**Comment by m7mad120059** [comment]
> Y

**Reply by goalend** [reply]
> > Y
> 
>
